intellij-ext/src/main/kotlin/com/metalbear/mirrord/MirrordListener.kt

```diff
@@ -66,33 +67,49 @@ class MirrordListener : ExecutionListener {
                         return@invokeLater super.processStartScheduled(executorId, env)
                     }
 
-                    val fileOpsCheckbox = JCheckBox("Enable File Operations")
-                    val remoteDnsCheckbox = JCheckBox("Enable Remote DNS")
-                    val ephemeralContainerCheckBox = JCheckBox("Enable Ephemeral Containers")
+                    val fileOps = JCheckBox("File Operations (default: true)")
```

Comment:
Wait is (default: true/false) necessary here?

Reply:
since we dont have toggles, i thought we should make it explicit. we can remove it, lmk

Reply:
Lets remove it, the default doesn't matter if the checkbox reflects the current value